src/GitTfs/GitTfs.cs

```diff
@@ -63,6 +74,47 @@ private void UpdateLoggerOnDebugging()
                 //consoleRule.DisableLoggingForLevel(LogLevel.Trace);
                 LogManager.ReconfigExistingLoggers();
             }
+
+            if (!String.IsNullOrEmpty(_globals.LogFilePath))
```

Comment:
PLease could you extract that in a (well named) method?

Reply:
Could you please expand on what you mean?  Do you want something like:
`if (_globals.LogFilePath.isSet())`
instead of
`if (!String.IsNullOrEmpty(_globals.LogFilePath))`